src/main.rs

```diff
@@ -1087,6 +1087,17 @@ fn main() {
                         binary with the Debug profile.")
                     .long("dbg-name")
                     .short("D"))
+                .arg(Arg::with_name("extension")
+                    .help("Builds the installer with a different extension")
+                    .long_help("By default the installer is created with a .msi extension. When \
+                        building a bundled package a .exe extensions is expected and the linker \
```

Comment:
"...When building a bundled package, a '.exe' file extension is expected by the WiX linker (light.exe). Use this option to change the file extension to `.exe` when needed."